### PR TITLE
Adding implementation for monitoring different Hedera Account Balances.

### LINKED ISSUE #474: Add the balance for each member in dashboard
At the moment we are displaying only one of the validator's balances in the `Operator account ID [0.0.XXX] ` section in the dashboard.

A new list with the addresses of the members - used for TX creation,  to be in the config, so we can have them as metrics.
accouint ID / validator name

ex for mainnet https://www.notion.so/limechain/Hashport-Deployment-Sequence-1fd2927a8a954c2c972f210f01c9146a

### PR DESCRIPTION
**Pull request by Psykepro**
Signed-off-by: Nikolay Nedkov <nikolay.nedkov@limechain.tech>

**Detailed description**:
Adding implementation for monitoring different Hedera Account Balances and adding health-check metric to the dashboard.

**Which issue(s) this PR fixes**:
Fixes #474 

**Special notes for your reviewer**:

**Checklist**
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: constants/prometheus.go [hunk 1]
```diff
@@ -25,8 +25,8 @@ const (
 	FeeAccountAmountGaugeHelp               = "Fee account amount."
 	BridgeAccountAmountGaugeName            = "bridge_account_amount"
 	BridgeAccountAmountGaugeHelp            = "Bridge account amount."
-	OperatorAccountAmountName               = "operator_account_amount"
-	OperatorAccountAmountHelp               = "Operator account amount."
+	AccountBalanceGaugeNamePrefix           = "account_balance_for_"
+	AccountBalanceGaugeHelpPrefix           = "Account balance for "
```
Comment (svetlio8) [inline constants/prometheus.go:29]: Looks like this one is not used, maybe it should be used at https://github.com/LimeChain/hashport-validator/blob/feature/balances-for-each-member/app/process/watcher/prometheus/watcher.go#L123

Comment (Psykepro) [reply constants/prometheus.go:29]: Ops yep I used the Name on the place where the Help needs to be :D

File: app/process/watcher/prometheus/watcher.go [hunk 2]
```diff
@@ -221,16 +251,21 @@ func (pw Watcher) setMetrics() {
...
 		if errPayerAcc == nil {
 			pw.payerAccountBalanceGauge.Set(pw.getAccountBalance(payerAccount))
 		}
 		if errBridgeAcc == nil {
 			pw.bridgeAccountBalanceGauge.Set(pw.getAccountBalance(bridgeAccount))
 		}
-		if errOperatorAcc == nil {
-			pw.operatorBalanceGauge.Set(pw.getAccountBalance(operatorAccount))
+
+		if len(pw.monitoredAccountsGauges) > 0 {
```
Comment (vectorman1) [inline app/process/watcher/prometheus/watcher.go:262]: unnecessary if

File: app/process/watcher/prometheus/watcher.go [hunk 3]
```diff
@@ -61,50 +73,68 @@ func NewWatcher(
...
-		prometheusService:         prometheusService,
-		logger:                    config.GetLoggerFor(fmt.Sprintf("Prometheus Metrics Watcher on interval [%s]", dashboardPolling)),
-		payerAccountBalanceGauge:  payerAccountBalanceGauge,
-		bridgeAccountBalanceGauge: bridgeAccountBalanceGauge,
-		operatorBalanceGauge:      operatorBalanceGauge,
-		assetsMetrics:             assetsMetrics,
-		assetsService:             assetsService,
+func initMonitoredAccountsGauges(configuration config.Config, prometheusService service.Prometheus) map[string]monitoredAccountsInfo {
+	monitoredAccountsGauges := make(map[string]monitoredAccountsInfo)
+	if len(configuration.Bridge.MonitoredAccounts) == 0 {
```
Comment (vectorman1) [inline app/process/watcher/prometheus/watcher.go:116]: unnecessary if

File: constants/regex.go [hunk 4]
```diff
@@ -0,0 +1,3 @@
+package constants
```
Comment (svetlio8) [inline constants/regex.go:1]: license is missing

**Review by vectorman1 — COMMENTED** [review]
I think these guard if's add unnecessary scope complexity when they don't do all that much - the assumption is that if a slice is empty a range over it will step over the for. 

Possibly down to style, idk, feel free to discuss

**Review by svetlio8 — DISMISSED** [review]

**Review by vectorman1 — DISMISSED** [review]

**Review by failfmi — APPROVED** [review]

File: docs/configuration.md [hunk 5]
```diff
@@ -51,6 +51,7 @@ Configuration for `config/bridge.yml`:
...
 |---------------------------------------------------------------|---------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `bridge.topic_id`                                             | ""      | The topic id, which the validators will use to monitor and submit consensus messages to.                                                                                                                                                                             |
+| `bridge.monitored_accounts[i]`                                | ""      | A mapping for all monitored accounts with prometheus where the `key` is the name of the account and `value` is the `account_id`.                                                                                                                                     |
```
Comment (failfmi) [inline docs/configuration.md:54]: Can we add that the values are Hedera Account IDs?

**Review by failfmi — APPROVED** [review]

**Review by svetlio8 — APPROVED** [review]

**Pull request merged**